GwentUtils.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/python3
+import json
+import re
+import os
+
+import xml.etree.ElementTree as xml
+
+LOCALES = ["en-US", "de-DE", "es-ES", "es-MX", "fr-FR", "it-IT", "ja-JP", "pl-PL", "pt-BR", "ru-RU", "zh-CN", "zh-TW"]
+
+def saveJson(filepath, data):
+    print("Saved JSON to: %s" % filepath)
+    with open(filepath, "w", encoding="utf-8", newline="\n") as f:
+        json.dump(data, f, sort_keys=True, indent=2, separators=(',', ': '))
+
+def cleanHtml(raw_html):
+    cleanr = re.compile('<.*?>')
+    cleantext = re.sub(cleanr, '', raw_html)
+    return cleantext
+
+class GwentDataHelper:
```

Comment:
There are no performance reasons (memory usage, caching, etc) for this class to exist.
Simply read and parse all the file-system data into a single `dict()`, and pass that into `CardData` (instead of making it call a bunch of methods and manually construct the `dict()` internally